GlobalConstantsService/data/ParticleList.txt

```diff
@@ -73,8 +73,8 @@
        113                   rho0                    same                                rho0       none          0       775.49   4.4057e-24
        115              a2(1320)0                    same                                a_20       none          0       1318.3  6.15151e-24
        130                 kaon0L                    same                                K_L0       none          0      497.614  5.08272e-08
-      -211                    pi-                    same                            pi_minus       none         -1    139.57018  2.58609e-08
-       211                    pi+                    same                             pi_plus       none          1    139.57018  2.58609e-08
+      -211                    pi-                    same                            pi_minus       none         -1    139.57018   2.6033e-08
```

Comment:
Thanks for updating this value.  It seems other values are out-of-date, including the muon lifetime.  It looks like this entire table needs to be refreshed.  That's outside the scope of this PR.

Reply:
yes, this was just a working fix to address an immediate issue